help_text/4c1560dca6a927e575ca4fc7f77516ad.md

```diff
@@ -0,0 +1,13 @@
+This error means that there are unencoded ampersands (&) in your markup.  HTML uses the ampersand character to start a character reference, and is thus a reserved symbol.  Instead of typing `&`, type the character reference `&amp;` instead.
+
+The following example is **not valid** because it does not use a character reference:
+
+```
+today & tomorrow
```

Comment:
Technically speaking this example _is_ valid, but will create a notice that you should use the full character reference. The error occurs when the ampersand is used in a place that it is ambiguous whether is it a literal or the delimeter (when there is no space before/after).